addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/hardware/callbacks/FritzAhaSetHeatingTemperatureCallback.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.avmfritz.internal.hardware.callbacks;
+
+import java.math.BigDecimal;
+
+import org.openhab.binding.avmfritz.internal.hardware.FritzahaWebInterface;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Callback implementation for updating heating values Supports reauthorization
+ * 
+ * @author Christoph Weitkamp
+ * 
+ */
+public class FritzAhaSetHeatingTemperatureCallback extends FritzAhaReauthCallback {
+    private final Logger logger = LoggerFactory.getLogger(getClass());

```

Comment:
Personally I prefer to pass the actual class, but since this pattern was like this for this specific binding its okay.